vault/logical_system.go

```diff
@@ -4551,7 +4551,12 @@ func (b *SystemBackend) pathInternalUIMountRead(ctx context.Context, req *logica
 		return errResp, logical.ErrPermissionDenied
 	}
 
-	filtered, err := b.Core.checkReplicatedFiltering(ctx, me, "")
+	var routerPrefix string
+	if strings.HasPrefix(me.APIPathNoNamespace(), credentialRoutePrefix) {
+		routerPrefix = credentialRoutePrefix
+	}
```

Comment:
Is the bug specific to `auth` mounts, or are other path prefixes also be affected?

Reply:
yeah, the secret engine mounts are working fine.